src/qofonosimwatcher.cpp

```diff
@@ -150,7 +161,8 @@ QOfonoSimWatcher::QOfonoSimWatcher(QObject *parent) :
 
 QOfonoSimWatcher::~QOfonoSimWatcher()
 {
-    // d_ptr is a child object, it gets deleted automatically
```

Comment:
I don't see how that could happen unless someone is processing events during object destruction.